### PR TITLE
Remove duplicate claim from jwt.

### PR DESCRIPTION
**Pull request by dfitchett**
Still having issues with request to bip. Their team recommended removing the duplicate expiration field from the JWT Claims. 

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java [hunk 1]
```diff
@@ -249,12 +251,25 @@ private <T extends BipPayloadResponse> T makeRequest(
               .statusMessage(HttpStatus.valueOf(bipResponse.getStatusCode().value()).name())
               .build();
     } catch (HttpStatusCodeException e) {
+
```
Comment (dfitchett) [inline svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java:255]: This block is to log the BIP Message that comes when their server responds with something other than a 2xx to our requests.

File: svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java [hunk 2]
```diff
@@ -198,7 +200,7 @@ private <T extends BipPayloadResponse> T makeRequest(
     try {
 
       HttpEntity<Object> httpEntity = new HttpEntity<>(requestBody, headers);
-      log.info("event=requestSent url={} method={}", url, method);
+      log.info("event=requestSent url={} method={} auth={}", url, method, headers.get("Authorization"));
```
Comment (dfitchett) [inline svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java]: This line will be reverted before redeploying to `prod`

File: svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiProps.java [hunk 3]
```diff
@@ -63,14 +62,17 @@ public String getAvailabilityUrl() {
    * @return Claims
    */
   public Claims toCommonJwtClaims(final String externalUserId, final String externalKey) {
-    Calendar cal = Calendar.getInstance();
-    cal.add(Calendar.MINUTE, 30);
-    Date expired = cal.getTime();
```
Comment (brostk) [inline svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiProps.java:68]: +1 for removing Calendar

File: svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java [hunk 4]
```diff
@@ -308,25 +324,16 @@ HttpHeaders getBipHeader(String externalUserId, String externalKey) throws BipEx
...
-    Date expiryDate = new Date(currentTimeMillis + 3600000);
-
     // Build the JWT
     String jwt =
         Jwts.builder()
             .claims(claims)
-            .issuer(issuer)
-            .subject("Claim")
-            .issuedAt(new Date(currentTimeMillis))
-            .expiration(expiryDate)
```
Comment (brostk) [inline svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java:329]: What's the motivation behind removing these fields on this Jwt builder but adding them in BipApiProps::toCommonJwtClaims?

Comment (dfitchett) [reply svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java:329]: Felt cleaner to just build all the claims in one place.

Comment (dfitchett) [reply svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java:329]: Honestly, I'd rip that all into another utility class to build a JWT with the BipApiProps class just used to populate values, but for now this is sufficient.

Comment (brostk) [reply svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java:329]: Fair, LGTM

**Review by brostk — APPROVED** [review]

**Review by lisac — APPROVED** [review]

**Pull request merged**